src/test/resources/features/highlighting/inspections/redundant_coma_inspection.feature

```diff
@@ -41,4 +41,54 @@ Feature: Inspection for redundant coma in the end of the line
       | rule_like  | section |
       | rule       | input   |
       | rule       | params  |
-      | checkpoint | input   |
\ No newline at end of file
+      | checkpoint | input   |
+
+  Scenario Outline: Fix redundant coma in the end of the line with two parameters
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    <rule_like> NAME:
+      <section>: "input1.txt", "input2.txt",
```

Comment:
Please merge this step with the previous one (they are almoste same), you could use pattern like:
```
    <rule_like> NAME:
      <section>: <content>,
```
And examples:
`| rule_like  | section | content | signature | `

Reply:
Changed